src/NuGet.Services.AzureSearch/AzureSearchJobDevelopmentConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Services.AzureSearch
+{
+    public class AzureSearchJobDevelopmentConfiguration
```

Comment:
I'm not super stoked by this name, given that config is only used by catalog2azuresearch and db2azuresearch. Please let me know if you have any suggestions here!